extensions/ql-vscode/src/remote-queries/gh-actions-api-client.ts

```diff
@@ -74,6 +74,18 @@ export async function getRemoteQueryIndex(
   };
 }
 
+export async function cancelRemoteQuery(
+  credentials: Credentials,
+  remoteQuery: RemoteQuery
+): Promise<void> {
+  const octokit = await credentials.getOctokit();
+  const { actionsWorkflowRunId, controllerRepository: { owner, name } } = remoteQuery;
+  const response = await octokit.request(`POST /repos/${owner}/${name}/actions/runs/${actionsWorkflowRunId}/cancel`);
```

Comment:
Optional suggestion 😺 

You could use the octokit "wrapper" for cancelling a workflow run from https://octokit.github.io/rest.js/v18#actions-cancel-workflow-run

Reply:
Hmmm...didn't know about that. But, I have a preference for keeping it closer to the actual rest API since it will be easier for us to look up the docs if we ever need to.